src/sidebar/services/test/frame-sync-test.js

```diff
@@ -458,6 +463,35 @@ describe('FrameSyncService', () => {
           [chapter1ann.$tag]: 'anchored',
         });
       });
+
+      describe('outside assignment warning', () => {
+        it('is not displayed if no focus filter is configured', async () => {
+          await connectEPUBGuest();
+          assert.isFalse(
+            guestRPC().call.calledWith('showOutsideAssignmentNotice'),
+          );
```

Comment:
Perhaps `assert.notCalledWith`?

```suggestion
          assert.notCalledWith(
            guestRPC().call,
            'showOutsideAssignmentNotice',
          );
```